thirdparty/xmlstream/xmlstream_p.h

```diff
@@ -1140,10 +1140,11 @@ bool XmlStreamReaderPrivate::parse()
             case '\n':
                 ++lineNumber;
                 lastLineStart = characterOffset + readBufferPos;
+                [[gnu::fallthrough]];
```

Comment:
So: brackets don't help, but comment `// fall through` is indeed respected and suppresses the warning. Might work with other comilers aswell, maybe?